x-pack/plugins/security_solution/public/timelines/components/timeline/styles.tsx

```diff
@@ -173,14 +173,24 @@ export const EventsTbody = styled.div.attrs(({ className = '' }) => ({
 
 export const EventsTrGroup = styled.div.attrs(({ className = '' }) => ({
   className: `siemEventsTable__trGroup ${className}`,
-}))<{ className?: string; eventType: Omit<TimelineEventsType, 'all'>; showLeftBorder: boolean }>`
+}))<{
+  className?: string;
+  eventType: Omit<TimelineEventsType, 'all'>;
+  isBuildingBlockType: boolean;
+  showLeftBorder: boolean;
+}>`
   border-bottom: ${({ theme }) => theme.eui.euiBorderWidthThin} solid
     ${({ theme }) => theme.eui.euiColorLightShade};
-  ${({ theme, eventType, showLeftBorder }) =>
-    showLeftBorder
+  ${({ theme, eventType, isBuildingBlockType, showLeftBorder }) =>
+    showLeftBorder && !isBuildingBlockType
       ? `border-left: 4px solid
     ${eventType === 'raw' ? theme.eui.euiColorLightShade : theme.eui.euiColorWarning}`
       : ''};
+  ${({ isBuildingBlockType, showLeftBorder }) =>
+    isBuildingBlockType
+      ? `background: repeating-linear-gradient(135deg, rgba(245, 167, 0, 0.2), rgba(245, 167, 0, 0.2) 1px, rgba(245, 167, 0, 0.05) 2px, rgba(245, 167, 0, 0.05) 10px);
```

Comment:
The below comment was meant for this occurrence (not the Building Block callout above):

> `127deg` here will result in the crosshatch lining up across multiple rows, which seems to be a bit less jarring and easier to focus on the content (n=1 here ;P).
> 
> ##### `135deg`
> <img alt="" width="500" src="https://user-images.githubusercontent.com/2946766/94869876-74726880-0403-11eb-8183-aef208d07008.png">
> 
> ##### `127deg`
> <img alt="" width="500" src="https://user-images.githubusercontent.com/2946766/94869928-9370fa80-0403-11eb-8045-6a5de85eca46.png">